Source/DataSources/CorridorGraphics.js

```diff
@@ -209,7 +212,15 @@ define([
          * @type {Property}
          * @default ClassificationType.BOTH
          */
-        classificationType : createPropertyDescriptor('classificationType')
+        classificationType : createPropertyDescriptor('classificationType'),
+
+        /**
+         * Gets or sets the zIndex Property specifying the ordering of the corridor.  Only has an effect if the coridor is static and neither height or exturdedHeight are specified.
+         * @memberof CorridorGraphics.prototype
+         * @type {ConstantProperty}
+         * @default 0
```

Comment:
All of the defaults say 0, but there's not actually a default is there?  It's just undefined.

Reply:
undefined === 0 for zIndex.  I `defaultValue(zIndex, 0)` in the geometry updaters